### PR TITLE
[15.0][MIG] intrastat_product

### PR DESCRIPTION
**Pull request by alexis-via**
Still have to fix Excel export of lines

### CODE REVIEW

File: intrastat_product/models/intrastat_region.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    _description = "Intrastat Region"
+    _sql_constraints = [
+        (
+            "intrastat_region_code_unique",
+            "UNIQUE(code, country_id)",  # TODO add company_id ?
+            "Code must be unique.",
+        )
+    ]
+
+    code = fields.Char(required=True)
```
Comment (flachica) [inline intrastat_product/models/intrastat_region.py:18]: Why dissapeared String of field?

File: intrastat_product/models/intrastat_transaction.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    _order = "code"
+    _sql_constraints = [
+        (
+            "intrastat_transaction_code_unique",
+            "UNIQUE(code, company_id)",
+            "Code must be unique.",
+        )
+    ]
+
+    code = fields.Char(required=True)
```
Comment (flachica) [inline intrastat_product/models/intrastat_transaction.py:21]: Why dissapeared String of field?

File: intrastat_product/models/intrastat_transport_mode.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+class IntrastatTransportMode(models.Model):
+    _name = "intrastat.transport_mode"
+    _description = "Intrastat Transport Mode"
+    _order = "code"
+    _sql_constraints = [
+        ("intrastat_transport_code_unique", "UNIQUE(code)", "Code must be unique.")
+    ]
+
+    code = fields.Char(required=True)
```
Comment (flachica) [inline intrastat_product/models/intrastat_transport_mode.py:17]: Why dissapeared String of some field on this model and others?

**Review by flachica — APPROVED** [review]

**Comment by rousseldenis** [comment]
@alexis-via I've added tests for Excel generation here https://github.com/OCA/intrastat-extrastat/pull/168

**Comment by victoralmau** [comment]
Please rebase from 14.0 to add the changes from https://github.com/OCA/intrastat-extrastat/pull/163

**Comment by rousseldenis** [comment]
@alexis-via

**Comment by MiquelRForgeFlow** [comment]
also add changes in https://github.com/OCA/intrastat-extrastat/pull/183

**Comment by victoralmau** [comment]
Superseed by: https://github.com/OCA/intrastat-extrastat/pull/187

**Pull request closed without merging**